indico/modules/events/papers/views.py

```diff
@@ -16,22 +16,28 @@
 
 from __future__ import unicode_literals
 
+from indico.modules.events.abstracts.views import _MathjaxMixin
```

Comment:
I was about to move it, but then I saw that we import it in the tracks module as well.